### DISCUSSION TITLE
Milvus Configuration File (milvus.yaml)

### DISCUSSION THREAD

**Discussion by alok1869** [milvus-io/milvus#30362] [Q&A and General discussion]
Dear All, 
Some time ago only, I started exploring milvus db. I'm looking the milvus.yaml file and checking the path of etcd defined in the milvus.yaml file, however I am not able to find the configured path . Below is etcd configuration of my milvus docker installation

# Related configuration of etcd, used to store Milvus metadata & service discovery.
etcd:
  endpoints: localhost:2379
  rootPath: by-dev # The root path where data is stored in etcd
  metaSubPath: meta # metaRootPath = rootPath + '/' + metaSubPath
  kvSubPath: kv # kvRootPath = rootPath + '/' + kvSubPath

Please help me where can I find by-dev and meta and kv folders in my docker file system.

**Comment by yhmo** [comment]
etcd.rootPath is the root prefix of the key to where Milvus stores data in etcd. It is managed by etcd, not a physical path. You can use etcd sdk/client to interact with etcd to browse the path.
